query06.sql

```diff
@@ -1,6 +1,20 @@
+-- Johnathan Clementi --
+-- MUSA 509 Geospatial Cloud Computing & Vizualization --
+-- HW #1 2021-09-19 --
+-- Query #6 --
+
 /*
   How many trips were shorter than 10 minutes?
 */
 
 -- Enter your SQL query here
-select ...
+with indego_trips_Q2_19and20 as (
+  select duration from indego_trips_2019_q2
+  union
+  select duration from indego_trips_2020_q2
+)
+
+select count(duration > 10) as tripsLess10Min
```

Comment:
Though it seems like `count(duration > 10)` _should_ do the right thing, it will not. `count(...)` will count all values that are not `null`. Since `duration > 10` always produces a value that's either `true` or `false` (unless `duration` is `null`), the value will always be counted. Do here, you're essentially counting all of the duration values, whether they are greater than or less than 10. The count is so low because of the `union` above, which ensured that we only had distinct duration values.